src/components/pages/order/adminPanel/BlocAdminPanelContent.js

```diff
@@ -2,32 +2,31 @@ import { useContext } from "react";
 import styled from "styled-components";
 import GlobalContext from "../../../../context/GlobalContext";
 import { theme } from "../../../../theme";
+import { getTabsConfig } from "./getTabsConfig";
 
 export default function BlocAdminPanelContent() {
-  const { isPanelReduce, panelTabIndex } = useContext(GlobalContext);
+  const { isPannelCollapsed, panelTabIndex } = useContext(GlobalContext);
+
+  const tabs = getTabsConfig(isPannelCollapsed, panelTabIndex);
+  const tabSelected = tabs.find((tab) => tab.index === panelTabIndex);
```

Comment:
```suggestion
  const tabSelected = tabs.find((tab) => tab.index === panelTabIndex);
```
essaie d'abstraire cette instruction dans une fonction à part : "tabs.find((tab) => tab.index === panelTabIndex)". On ne veut pas connaitre l'**implémentation**. On veut juste l'**intention**.